gulpfile.js

```diff
@@ -13,177 +13,179 @@ USWDS SASS GULPFILE
 
 const autoprefixer = require("autoprefixer");
 const csso = require("postcss-csso");
-const gulp = require("gulp");
-const pkg = require("./node_modules/uswds/package.json");
+const { src, dest, series, parallel, watch } = require("gulp");
+const pkg = require("../uswds/package.json").version;
```

Comment:
If we recommend copying USWDS this will need to be updated. But this path is required for running tasks without any setup.

Example:
```bash
npx gulp buildSass --gulpfile ./node_modules/uswds-gulp/gulpfile.js --cwd .
```